src/test/java/checker/MiLoGCheckerTimeOverlapTest.java

```diff
@@ -199,7 +199,7 @@ public void testMultipleEntriesWithOverlap() {
         checker.checkTimeOverlap();
         
         ////Expectation
-        String error = MiLoGChecker.MiLoGCheckerErrorMessage.TIME_OVERLAP.getErrorMessage(date0);
+        String error =  MiLoGChecker.MiLoGCheckerErrorMessageProvider.TIME_OVERLAP.getErrorMessage(date0);
```

Comment:
I think there happend a little mistake, namely a space in front of every occurrence, when you changed the name of the error providing class.

Reply:
Thank you for noticing, will fix it!